packages/avatar-group/theme/lumo/vaadin-avatar-group.js

```diff
@@ -1,4 +1,4 @@
-import './vaadin-avatar.js';
+import '@vaadin/avatar/theme/lumo/vaadin-avatar.js';
```

Comment:
Let's move the import below `import './vaadin-avatar-group-styles.js';` to be placed after external imports to follow the same order as we generally use in both `src` and `theme` folders of components.